### ISSUE TITLE
High contrast theme for outdoor use by small group

### ISSUE THREAD

**Issue by 90084** [biblesoapopera/demo#47]
Would be good to have an optional high contrast theme - I expect this will often be viewed by 2-4 people huddled around a mobile, often outdoors. 
Low priority.

**Comment by superdweebie** [comment]
I totally agree. I quite like how the layout of the new ui is working, but the more I work with it the more I am getting sick of the burnt orange colour! I'm going to put this in the 0.2.0 milestone.

**Comment by 90084** [comment]
ha, I personally like the orange...

**Comment by superdweebie** [comment]
I'd prefer one really solid theme that just works. However, we could add a theme switcher button to the ui for high contrast / standard. Thoughts?

**Comment by 90084** [comment]
Yep, if we can pull it off, one theme that works in all use contexts is best.
